moped-editor/src/utils/activityLogFormatters/mopedMilestonesActivity.js

```diff
@@ -0,0 +1,46 @@
+import EventNoteIcon from "@material-ui/icons/EventNote";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatMilestonesActivity = (change, milestoneList) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_proj_milestones"];
+
+  const changeIcon = <EventNoteIcon />;
+  let changeDescription = "Project milestone updated";
+  let changeValue = "";
+
+  // add a new milestone
+  if (change.description.length === 0) {
+    changeDescription = "Added a new milestone: ";
+    changeValue = milestoneList[change.record_data.event.data.new.milestone_id];
+
+    return { changeIcon, changeDescription, changeValue };
+  }
+
+  // delete an existing milestone
+  if (change.description[0].field === "is_deleted") {
+    changeDescription = "Deleted the milestone: ";
+    changeValue = milestoneList[change.record_data.event.data.new.milestone_id];
+
+    return { changeIcon, changeDescription, changeValue };
+  }
+
+  // Multiple fields in the moped_proj_funding table can be updated at once
```

Comment:
should this say milestones table?

Reply:
eek my impatient copy and pasting habits -- thanks ill fix it